open-build-service-api/src/lib.rs

```diff
@@ -408,6 +484,100 @@ impl<'a> PackageBuilder<'a> {
         self.client.request(u).await
     }
 
+    pub async fn source_file(&self, file: &str) -> Result<Vec<u8>> {
```

Comment:
assuming this is downloading files it probably should return a stream object rather then just the raw bytes.